### ISSUE TITLE
[NOT A ISSUE] Question about CSS

### ISSUE THREAD

**Issue by z3xddd** [kristopolous/BOOTSTRA.386#100]
Hello @kristopolous,

I would like to congratulate you and everyone who contributed to this project, it was awesome and I hope to be able to support you.

I have a question about the rendering that occurs on this page: https://kristopolous.github.io/BOOTSTRA.386/index.html

I would like to know what was implemented for the rendering of the page to be done little by little, as I would like to implement it on my website.

Greetings,
@z3xddd

**Comment by kristopolous** [comment]
no, it's done in javascript through some clever hack.  Look at the loading() function here: https://kristopolous.github.io/BOOTSTRA.386/assets/js/bootstrap-386.js
